administrator-manual/en/base_system2.rst

```diff
@@ -261,3 +261,49 @@ The following pages are always accessible to all users:
 * dashboard
 * applications
 * terminal
+
+Two-factor authentication (2FA)
+===============================
+
+Two-factor authentication (2FA) is an extra layer of security required to access the new Server Manager.
```

Comment:
```suggestion
Two-factor authentication (2FA) is an extra layer of security that can be added to access the new Server Manager.
```

Reply:
I do not like it.

What about:
```
Two-factor authentication (2FA) adds extra layer of security required ro access the new Server Manager.
```

@filippocarletti ?

Reply:
Pay attention to the "required ro" typo :)

I think Filippo's idea is to suggest that the 2FA is not mandatory. The extra layer is optional.

Reply:
```suggestion
Two-factor authentication (2FA) can be used to add an extra layer of security required to access the new Server Manager.
```